be/src/storage/persistent_index.cpp

```diff
@@ -60,6 +60,12 @@ constexpr size_t kPackSize = 16;
 constexpr size_t kBucketSizeMax = 256;
 constexpr size_t kFixedMaxKeySize = 128;
 constexpr size_t kBatchBloomFilterReadSize = 4ULL << 20;
+/*
+The introduction of this magic number serves two purposes:
+1. To detect endianness mismatches in cross-platform scenarios
+2. To identify the new snapshot encoding format
+*/
```

Comment:
```suggestion
// The introduction of this magic number serves two purposes:
// 1. To detect endianness mismatches in cross-platform scenarios
// 2. To identify the new snapshot encoding format
```
Prefer comment starts with`//`